Cargo.lock

```diff
@@ -213,8 +213,8 @@ dependencies = [
  "glob",
  "indoc",
  "lazy_static",
- "libcnb",
- "libherokubuildpack",
+ "libcnb 0.21.0",
```

Comment:
(for the future) It would be good to remove the dependency on Ruby commons to prevent double-including different versions of libcnb.rs like this

Reply:
Noted. I'd really like to tackle the inconsistent output across all these CNBs soon but, right now, this doesn't seem to be causing a problem.